source/develop/release-management/features/infra/link-following.html.md

```diff
@@ -0,0 +1,114 @@
+---
+title: API Link Following
+category: feature
+authors: oliel
+wiki_category: Feature
```

Comment:
wiki_* headers DO NOT exist, and NEVER existed

Reply:
Please explain this comment